tasks/src/test/java/com/redhat/parodos/tasks/TibcoWorkFlowTaskTest.java

```diff
@@ -0,0 +1,73 @@
+// @formatter:off
+package com.redhat.parodos.tasks;
+
+import com.redhat.parodos.workflow.context.WorkContextDelegate;
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.jms.JMSException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class TibcoWorkFlowTaskTest {
+
+	String caFile = "/cafile";
+	String url = "ssl://localhost:7222";
+	String username = "username";
+	String passowrd = "password";
+	String topic = "topic";
+	String message = "test message";
+	Tibjms tibjms = Mockito.mock(Tibjms.class);
+	TibcoWorkFlowTask task = new TibcoWorkFlowTask(tibjms, url, caFile, username, passowrd);
+
+	@Before
+	public void before() {
```

Comment:
To be consistent whit other tests, this method should be renamed in `setUp()`

Reply:
@gciavarrini fixed